src/promptflow-devkit/promptflow/_sdk/_utilities/general_utils.py

```diff
@@ -975,6 +975,25 @@ def create_temp_flex_flow_yaml_core(entry: Union[str, PathLike, Callable], code:
         code = Path(code)
         if not code.exists():
             raise UserErrorException(f"Code path {code.as_posix()} does not exist.")
+    return entry, code
+
+
+def create_flex_flow_yaml_in_target(entry: Union[str, PathLike, Callable], target_dir: str, code: Path = None):
+    """
+    Generate a flex flow yaml in target folder. The code field in the yaml points to the original flex yaml flow folder.
+    """
+    from promptflow._utils.flow_utils import dump_flow_dag_according_to_content
+
+    logger.info("Create temporary entry for flex flow.")
+    entry, code = resolve_entry_and_code(entry, code)
+    flow_dag = {"entry": entry, "code": str(code.resolve())}
```

Comment:
if code is same as target_dir, we can log ".' here

Reply:
updated